R/database.R

```diff
@@ -1332,7 +1398,29 @@ get_manta_sv = function(these_sample_ids,
   if(from_flatfile){
     all_sv = get_combined_sv(projection = projection)
 
-    all_meta = get_gambl_metadata()
+    #sample IDs are provided, metadata is not
```

Comment:
Nice catch, I did not think of all the scenarios - thanks for catching these and handling them!

Reply:
Yeah, no worries!